src/reports.js

```diff
@@ -22,11 +28,11 @@ function sendReports (job, flag, done) {
   const channelMap = {}
 
   if (flag === 'dailyReport') {
-    from = moment().subtract(1, 'days').startOf('day').toDate()
-    to = moment().subtract(1, 'days').endOf('day').toDate()
+    from = moment().subtract(1, 'days').startOf('day').utcOffset(utcOffset).format("Do MMM YYYY HH:mm:ss Z")
```

Comment:
@bradsawadye Please could you rename these variables - they're a bit vague - to what? From who? The Variable should be self documenting